lib/spec_file_generator.js

```diff
@@ -19,10 +19,17 @@ function sorted(items) {
 }
 
 function dependenciesToSources(deps) {
+  deps.sort((a, b) => {
+    if (a[0] === b[0]) {
+      return a[1].localeCompare(b[1]);
+    } else {
+      return a[0].localeCompare(b[0]);
+    }
+  });
   const sources = deps.map((dependency, index) => {
     return `Source${index}: ${npmUrl(dependency[0], dependency[1])}`;
   });
-  sources.push(`Source${deps.length}: ${getCacheFilename('%{name}', '%{version}')}`);
+  sources.push(`Source${deps.length}: ${getCacheFilename('nodejs-%{npm_name}', '%{version}')}`);
```

Comment:
We have a function for this: `getRpmPackageName(npm_module.name)` is what's used in `generateSpecFile`: https://github.com/dLobatog/npm2rpm/blob/4a4ca951fc6ec3350a34ca05e6ca4b889701f384/lib/npm_helpers.js#L5-L14

Might make sense to change `getCacheFilename` to call `getRpmPackageName` since that's also done here: https://github.com/dLobatog/npm2rpm/blob/4a4ca951fc6ec3350a34ca05e6ca4b889701f384/bin/npm2rpm.js#L104

However, in the actual spec I think I'd still prefer the macro `%{version}` so updating the spec is a smaller diff.